src/gui/dialog.hpp

```diff
@@ -81,6 +82,24 @@ class Dialog
     MenuManager::instance().set_dialog(std::move(dialog));
   }
 
+  // the callback is called regardless the state(enabled/disabled) of confirmation dialog
+  static void show_confirmation(const std::string& text, const std::function<void ()>& callback)
+  {
+    if (g_config->confirmation_dialog)
+    {
+      std::unique_ptr<Dialog> dialog(new Dialog);
+      dialog->set_text(text);
+      dialog->clear_buttons();
+      dialog->add_cancel_button(_("No"));
+      dialog->add_default_button(_("Yes"), callback);
```

Comment:
UI/UX: Change the order: First the yes, afterwards the "no" button (it's like that in nearly every other dialogue).